packages/sync-actions/src/product-actions.js

```diff
@@ -273,6 +274,7 @@ function _buildVariantPricesAction(
         // still has other values, otherwise simply return
         const filteredPrice = { ...price }
         delete filteredPrice.discounted
+        delete filteredPrice.custom
```

Comment:
Is this necessary so that we still create `changePrice` update actions?

Reply:
i think i wont be right to create `changePrice` action for `custom` prop, since we now create specific action for `custom` props.
i even tried it with platform api to send this action 
`{
  "action": "changePrice",
  "priceId": "111",
  "price": {
    "custom": {
      "type": {
        "id": "2222"
      },
      "fields": {
        "promotionDiscountType": "Tiefpreis",
        "promotionRegularPrice": 8995,
        "promotionValidToExclusive": "SKU-111",
        "promotionValidFromInclusive": "2018-10-02T00:00:35.570Z"
      }
    },
  }
}`

the respond were like they didnt even add anything
and i think as they mentioned on doc, this field `custom` for price can only change with these two actions `setProductPriceCustomType` `setProductPriceCustomField`

thats why i exclude it here, and pass it to `actionsMapPricesCustom`

Reply:
Please ignore what i commented above, i need to make sure of sth

Reply:
you totally right, thanks :), i even added test to make sure that custom prop wont be deleted while action `changePrice`